src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1201,9 +1201,9 @@
         "FEATURE": {
             "bgp": {
                 "auto_restart": "enabled",
-                "has_global_scope": "false",
-                "has_per_asic_scope": "true",
-                "has_timer": "false",
+                "has_global_scope": "False",
+                "has_per_asic_scope": "True",
+                "has_timer": "False",
```

Comment:
I have added these changes with "False" and "True" with a capital letter to make it clear that this option can also be used and it is valid.